stetho/src/main/java/com/facebook/stetho/inspector/protocol/module/Database.java

```diff
@@ -244,4 +272,34 @@ public ExecuteSQLResponse handleUpdateDelete(int count) throws SQLiteException {
     @JsonProperty(required = true)
     public int code;
   }
+
+  public static abstract class DatabasePeer {
+
+    protected Context mContext;
+
+    public DatabasePeer(Context context) {
+      mContext = context;
+    }
+
+    protected abstract void onRegistered(JsonRpcPeer peer);
```

Comment:
Please check the latest commit. As you suggested, implementor shouldn't implement this class anymore.